### PR TITLE
CLI begin and end messages

### PR DESCRIPTION
**Pull request by anthonyburdi**
Changes in this PR
- Add a usage stats message to the beginning and end of each CLI command for the v3 API except for `suite`.
- Add a "cancelled" boolean for CLI commands that are cancelled during operation.

TODO:

- [x] Checkpoint
- [x] Datasource
- [x] Docs
- [x] Init (NA)
- [x] Project
- [x] Store
- [ ] Suite - potentially done in a separate PR against the fork
- [x] ~~Validation Operator~~

### CODE REVIEW

File: great_expectations/cli/checkpoint.py [hunk 1]
```diff
@@ -166,43 +169,49 @@ def _get_notebook_path(context, notebook_name):
...
 @checkpoint.command(name="delete")
 @click.argument("checkpoint")
 @click.pass_context
 def checkpoint_delete(ctx, checkpoint):
     """Delete a Checkpoint."""
-    usage_event: str = "cli.checkpoint.delete"
     context: DataContext = ctx.obj.data_context
+    usage_event_end: str = toolkit.send_cli_begin_usage_event(
+        context=context, usage_event_prefix=f"{ctx.obj.usage_stats_prefix}.delete"
+    )
```
Comment (alexsherstinsky) [inline great_expectations/cli/checkpoint.py]: ❤️

File: great_expectations/core/usage_statistics/schemas.py [hunk 2]
```diff
@@ -331,6 +331,29 @@
     "additionalProperties": False,
 }
 
+cancelled_cli_payload_schema = {
```
Comment (alexsherstinsky) [inline great_expectations/core/usage_statistics/schemas.py:334]: Now I really like this!!!

File: great_expectations/cli/store.py [hunk 3]
```diff
@@ -19,18 +19,35 @@ def store(ctx):
...
 @click.pass_context
 def store_list(ctx):
     """List active Stores."""
     context = ctx.obj.data_context
-    stores = context.list_active_stores()
-    cli_message(f"{len(stores)} active Stores found:")
-    for store in stores:
-        cli_message("")
-        cli_message_dict(store)
+    usage_event_end: str = ctx.obj.usage_event_end
```
Comment (alexsherstinsky) [inline great_expectations/cli/store.py:36]: Where is the `begin` event?..

File: tests/cli/test_project.py [hunk 4]
```diff
@@ -42,6 +49,21 @@ def test_project_check_on_valid_project_says_so(
     assert "Checking your config files for validity" in result.output
     assert "Your config file appears valid" in result.output
     assert result.exit_code == 0
+
```
Comment (alexsherstinsky) [inline tests/cli/test_project.py:52]: Thank you for adding these assertions!!! ❤️

**Review by alexsherstinsky — CHANGES_REQUESTED** [review]
I would like to approve this after we briefly discuss the items mentioned in my comments.

File: great_expectations/cli/store.py (hunk 3 above)
Comment (anthonyburdi) [reply great_expectations/cli/store.py:36]: The begin events are all in the base commands! That way we never forget to add them for new CLI commands :) And also a bit more DRY. E.g. look at `def store()`

**Review by alexsherstinsky — APPROVED** [review]
LGTM once all checks pass.

**Pull request merged**